demo/.gitignore

```diff
@@ -1,5 +1,5 @@
 # ignore our import cache
-.import
+.godot
```

Comment:
@paddy-exe you'll want to do this on your branch and delete your `.godot` folder and update the branch. You checked in a whole bunch of cache files :)

Reply:
Ah yeah that makes sense